www-apps/miniflux/miniflux-2.0.36.ebuild

```diff
@@ -0,0 +1,740 @@
+	"google.golang.org/grpc v1.19.0/go.mod"
+	"google.golang.org/grpc v1.20.1/go.mod"
+	"google.golang.org/grpc v1.21.0/go.mod"
+	"google.golang.org/grpc v1.21.1/go.mod"
+	"google.golang.org/grpc v1.23.0/go.mod"
+	"google.golang.org/grpc v1.25.1/go.mod"
+	"google.golang.org/grpc v1.26.0/go.mod"
+	"google.golang.org/grpc v1.27.0/go.mod"
+	"google.golang.org/grpc v1.27.1/go.mod"
+	"google.golang.org/grpc v1.28.0/go.mod"
+	"google.golang.org/grpc v1.29.1/go.mod"
+	"google.golang.org/grpc v1.30.0/go.mod"
+	"google.golang.org/grpc v1.31.0/go.mod"
+	"google.golang.org/protobuf v0.0.0-20200109180630-ec00e32a8dfd/go.mod"
+	"google.golang.org/protobuf v0.0.0-20200221191635-4d8936d0db64/go.mod"
+	"google.golang.org/protobuf v0.0.0-20200228230310-ab0ca4ff8a60/go.mod"
+	"google.golang.org/protobuf v1.20.1-0.20200309200217-e05f789c0967/go.mod"
+	"google.golang.org/protobuf v1.21.0/go.mod"
+	"google.golang.org/protobuf v1.22.0/go.mod"
+	"google.golang.org/protobuf v1.23.0/go.mod"
+	"google.golang.org/protobuf v1.23.1-0.20200526195155-81db48ad09cc/go.mod"
+	"google.golang.org/protobuf v1.24.0/go.mod"
+	"google.golang.org/protobuf v1.25.0/go.mod"
+	"google.golang.org/protobuf v1.26.0-rc.1/go.mod"
+	"google.golang.org/protobuf v1.26.0"
+	"google.golang.org/protobuf v1.26.0/go.mod"
+	"gopkg.in/alecthomas/kingpin.v2 v2.2.6/go.mod"
+	"gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127/go.mod"
+	"gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15/go.mod"
+	"gopkg.in/errgo.v2 v2.1.0/go.mod"
+	"gopkg.in/resty.v1 v1.12.0/go.mod"
+	"gopkg.in/square/go-jose.v2 v2.5.0"
+	"gopkg.in/square/go-jose.v2 v2.5.0/go.mod"
+	"gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7/go.mod"
+	"gopkg.in/yaml.v2 v2.2.1/go.mod"
+	"gopkg.in/yaml.v2 v2.2.2/go.mod"
+	"gopkg.in/yaml.v2 v2.2.4/go.mod"
+	"gopkg.in/yaml.v2 v2.2.5/go.mod"
+	"gopkg.in/yaml.v2 v2.3.0/go.mod"
+	"gopkg.in/yaml.v2 v2.4.0/go.mod"
+	"gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c"
+	"gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c/go.mod"
+	"honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod"
+	"honnef.co/go/tools v0.0.0-20190106161140-3f1c8253044a/go.mod"
+	"honnef.co/go/tools v0.0.0-20190418001031-e561f6794a2a/go.mod"
+	"honnef.co/go/tools v0.0.0-20190523083050-ea95bdfd59fc/go.mod"
+	"honnef.co/go/tools v0.0.1-2019.2.3/go.mod"
+	"honnef.co/go/tools v0.0.1-2020.1.3/go.mod"
+	"honnef.co/go/tools v0.0.1-2020.1.4/go.mod"
+	"mvdan.cc/xurls/v2 v2.4.0"
+	"mvdan.cc/xurls/v2 v2.4.0/go.mod"
+	"rsc.io/binaryregexp v0.2.0/go.mod"
+	"rsc.io/pdf v0.1.1/go.mod"
+	"rsc.io/quote/v3 v3.1.0/go.mod"
+	"rsc.io/sampler v1.3.0/go.mod"
+)
+
+go-module_set_globals
+
+DESCRIPTION="Minimalist and opinionated feed reader"
+HOMEPAGE="https://miniflux.app https://github.com/miniflux/v2"
+SRC_URI="https://github.com/${PN}/v2/archive/${PV}.tar.gz -> ${P}.tar.gz
+		${EGO_SUM_SRC_URI}"
+
+LICENSE="Apache-2.0 BSD BSD-2 MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
```

Comment:
I tend to prefer to not list blank variables when unnecessary: see [here](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3).